### PR TITLE
Feature mulit assets indicator

### PR DESCRIPTION
**Pull request by yidong72**
to compute features manually, I need to compute indicators for all assets. 
These changes allow me to compute indicators for all assets efficiently.

### CODE REVIEW

File: notebook/cuIndicator/indicator_demo.ipynb [hunk 1]
```diff
@@ -96,15 +96,15 @@
   },
   {
```
Comment (avolkov1) [inline notebook/cuIndicator/indicator_demo.ipynb]: Cell 4 with `get_figure` function is massive. Can that be refactored into a [factory ](https://realpython.com/factory-method-python/) and into a separate module? Then import from the module.

File: tests/unit/test_multi_assets_indicator.py [hunk 2]
```diff
@@ -0,0 +1,787 @@
...
+        high_pdf['High'] = high_array[half:]
+        high_pdf['Low'] = low_array[half:]
+        high_pdf['Volume'] = volume_array[half:]
+
+        self._pandas_data = pdf
+        self._cudf_data = df
+        self._plow_data = low_pdf
+        self._phigh_data = high_pdf
+        warnings.simplefilter('ignore', category=ImportWarning)
+        warnings.simplefilter('ignore', category=DeprecationWarning)
```
[the remark this answers is not in the document]
Comment (yidong72) [reply tests/unit/test_multi_assets_indicator.py]: I did that

File: notebook/cuIndicator/indicator_demo.ipynb (hunk 1 above)
Comment (yidong72) [reply notebook/cuIndicator/indicator_demo.ipynb]: It is a different task. Let's postpone this for the future.

**Comment by yidong72** [comment]
We can make a different issue to clean up the notebooks.

**Pull request merged**